packages/controllers/src/permissions/README.md

```diff
@@ -142,6 +160,17 @@ const permissionSpecifications = {
       return context.getSecret(secretName);
     },
   },
+
+  // This is an endowment.
+  secretEndowment: {
```

Comment:
Should it be possible for the endowment to be unrestricted? That would mean that all endowments ever used would be specified in one place only - in PermissionController.

Reply:
Yeah, at the time of writing, I can't think of any _unrestricted_ endowment that would have to be specified. In the context of SES, for instance, an unrestricted endowment would just be available by default.